### ISSUE TITLE
set_git_credentials_cach() throws a usage warning

### ISSUE THREAD

**Issue by FvD** [ixpantia/ixplorer#51]
```
ixplorer::set_git_credentials_cache()
usage: git config [<options>]

Config file location
    --global              use global config file
    --system              use system config file
    --local               use repository config file
    --worktree            use per-worktree config file
    -f, --file <file>     use given config file
    --blob <blob-id>      read config from given blob object
```

**Comment by nayib-ixpantia** [comment]
Wrong syntaxis at set_git_credentials_cache function, `cache --timeout` is not enclosed between apostrophes ('')
```
 "git config --global credential.helper cache --timeout=",
      hours * 60 * 60))
```
According to this [documentation](https://git-scm.com/docs/git-credential-cache), the command is: 
`git config credential.helper 'cache --timeout=300'`

So, the proper way to correct this should be:
```
 "git config --global credential.helper 'cache --timeout=",
      hours * 60 * 60, "'"))
```

**Comment by nayib-ixpantia** [comment]
Also, the instruction show.output.on.console = FALSE should be added as follows:
```
 "git config --global credential.helper 'cache --timeout=",
      hours * 60 * 60, "'"), show.output.on.console = FALSE)
```

**Comment by andreavargasmon** [comment]
closed with #115
